File: code/coding_pratice_Interview/Kth_largest_element/Kth_largest_element.py
def Kth_largest_element(arr):
    big = arr[0]
    for i in range(len(arr)):
        if(arr[i] > big):
            big = arr[i]
    
    print("Kth largest element is: ", big)

Kth_largest_element([1,2,4,5,45,23,56])

File: code/coding_pratice_Interview/Kth_largest_element/Kth_largest_element.js
const Kth_largest_element = (arr) =>{
    let kth = 0;
    for(let i = 0; i < arr.length; i++){
        if(arr[i] > kth){
            kth = arr[i]
        }
    }
    console.log("Kth_largest_element is: ", kth);
    
}
Kth_largest_element([234,236,89,23,45,123])